config/routes.rb

```diff
@@ -18,6 +18,7 @@
     namespace :v0 do
       resources :tokens, only: [:create]
       get 'manifests/search', to: 'manifests#search'
+      post 'manifests/validate', to: 'manifests#validate'
```

Comment:
oh, the `1` is just a true value.

If we could have arbitrary HTTP verbs, this would be ideal:

`VALIDATE /manfests  < manifest.json`

but alas, we are constrained.